website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: intro to flint
+title: Introduction to FLINT
+author: Ruchi Pakhle
+author_url: https://github.com/Ruchip16
+author_title: Passionate programmer, coder & a web-developer 
+tags: [Intro to Flint, docusaurus]
+description: Introduction to Flint
+---
+<!-- Summary -->
+
+**The Full Lands Integration Tool (FLINT)** is moja global’s flagship software. FLINT is a modular and highly flexible open-source software technology designed to meet the needs of governments, the private sector and other users in measuring, reporting and verifying (MRV) – or forecasting – greenhouse gas emissions and removals from forestry, agriculture and other land uses (AFOLU).
+ 
+<!--truncate-->
+
+### What is FLINT?
+FLINT stands for (Full Lands Integration Tool) which is a C++ framework that combines the data acquired from the satellites & ground in a way that meets the requirements of the policy. FLINT is an *open-source library* maintained under **moja-global**, a project under the **LINUX Foundation** 
+
+### Why FLINT?
+FLINT enables all countries to develop and operate advanced systems. It is based on more than 20 years of experience in building and operating integrated tools in Australia and Canada. It is a universal platform with a modular structure, allowing countries to attach any kind of models or data to build a country-specific system. The platform handles complex computer science tasks, such as storing and processing large data sets, allowing users to focus on monitoring, reporting, and scenario analysis.
```

Comment:
Hey @Ruchip16 - great work! 

Can we add some more context to this paragraph:
> FLINT enables all countries to develop and operate advanced systems for monitoring land use and land-use change. It is based on more than 20 years of experience in building and operating integrated tools for national greenhouse-gas (GHG) invenotries in Australia and Canada. The FLINT is a universal platform with a modular structure, allowing countries to attach any kind of models or data to build a country-specific system. The platform handles complex computer science tasks, such as storing and processing large data sets, allowing users to focus on monitoring, reporting, and scenario analysis.

Reply:
Sure! Can you give me some points so that I can elaborate on them

Reply:
I have made some changes to the quoted text in my comment. You can just copy and paste it!

Reply:
Cool! Thank you :)